spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_target_vm_spec.rb

```diff
@@ -66,6 +66,32 @@
     assert_storage(storage, vm)
   end
 
+  context 'vm migration' do
+    before(:each) do
```

Comment:
a similar `before` block is at the beginning of this spec. Do you really need both?
How about moving the reset of this file into its own context with individual `before` blocks.
Or even better get rid of them, as they tend to be overseen in larger files